xicam/plugins/__init__.py

```diff
@@ -170,11 +169,11 @@ def hot_reload(self):
     def _discover_plugins(self):
         self.state = State.DISCOVERING
         # for each plugin type
-        for type_name in self.plugin_types.keys():
+        for type_name, plugin_type in self.plugin_types.items():
 
             # get all entrypoints matching that group
-            group = entrypoints.get_group_named(f"xicam.plugins.{type_name}")
-            group_all = entrypoints.get_group_all(f"xicam.plugins.{type_name}")
+            group = entrypoints.get_group_named(f"{getattr(plugin_type,'entrypoint_prefix', 'xicam.plugins.')}{type_name}")
```

Comment:
I was a bit confused here before I noticed the new `entrypoint_prefix` attribute in the `PluginType` class. I think it is smart to embed the prefix itself into the `PluginType` derived classes